prediction-applier/src/main/java/org/alfresco/hxi_connector/prediction_applier/auth/PredictionApplierHxAuthClient.java

```diff
@@ -51,4 +65,44 @@ public AuthenticationResult authenticate(String tokenUri, ClientRegistration cli
     {
         return super.authenticate(tokenUri, clientRegistration);
     }
+
+    @Retryable(retryFor = EndpointServerErrorException.class,
+            maxAttemptsExpression = "#{@hxInsightProperties.hylandExperience.authentication.retry.attempts}",
+            backoff = @Backoff(delayExpression = "#{@hxInsightProperties.hylandExperience.authentication.retry.initialDelay}",
+                    multiplierExpression = "#{@hxInsightProperties.hylandExperience.authentication.retry.delayMultiplier}"))
+    @Override
+    public AuthenticationResult authenticate(String clientRegistrationId)
```

Comment:
This method will be moved to the parent class and will be simplified to trigger camel authentication rout with needed parameters.